Sprint-3/implement/get-angle-type.js

```diff
@@ -10,18 +10,58 @@
 // When the angle is exactly 90 degrees,
 // Then the function should return "Right angle"
 
+function getAngleType(degrees){
+  if (degrees === 90){
+    return ("Right angle");
+  }
+  
+}
+test ("angle is exactly 90 degrees", function() {
+  expect(getAngleType(90)).toEqual("Right angle");
+});
+
 // Identify Acute Angles:
 // When the angle is less than 90 degrees,
 // Then the function should return "Acute angle"
 
+function getAngleType(degrees){
+  if (degrees < 90){
```

Comment:
The function could potentially recognize an angle of, for instance, -10° as an acute angle. How could this be rectified?